docs/reference/css_guidelines.rst

```diff
@@ -0,0 +1,151 @@
+Contributing CSS to Wagtail
```

Comment:
I've made a new section in the docs. Much better keeping all that stuff together.